src/components/Compare/CompareTableRow.tsx

```diff
@@ -85,7 +85,9 @@ const CompareTableRow = (props: {
           </div>
           <div>
             {locationName[0]}{' '}
```

Comment:
So just thinking a bunch about this code, and have a thought/suggestion, I think this logic for generating the location name in the row could be it's own component with a bit less shared logic, could even be a structure like:
```
const CompareLocationName = ({region: Region, showStateCode: boolean}) => {
  if (region instanceof State) {
      return (<div>region.fullName</div>);
  } else if (region instanceof County) {
     return (
         const splitLocationName = splitLocationName(region.abbrev);
         return (
           <div>
              {locationName[0]}{' '}
              <LocationSuffix>{locationName[1]}</LocationSuffix>
              {showStateCode && ... }
           </div>
         ) else if (region instanceof MetroArea) {
             ...
         }
         fail('unsupported region');
    );
  }

}

```